### PR TITLE
Awesome books with ES6

### PR DESCRIPTION
**Pull request by Adams-Ghub**
- Create books module
- Add Luxon time library to display time
- Re-structure all functions to arrow functions
- Update README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header>
+      <p class="page-title">Awesome Books</p>
+      <ul class="menu">
+        <li class="separator"><a href="#" id="list">List</a></li>
+        <li class="separator"><a href="#" id="add">Add new</a></li>
+        <li><a href="#" id="contact">Contact</a></li>
+      </ul>
```
Comment (AmaduKamara) [inline index.html:17]: - [ ] Clicking on any of the links expects to show a section for that link. Kindly make sure the app work fine, routing and all other features as well. 👍🏽

File: index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+const showPage = (show, hide) => {
+  const showElement = document.querySelector(show);
+  showElement.classList.remove('hide');
+  for (let i = 0; i < Object.keys(hide).length; i++) {
+    const hideElement = document.querySelector(hide[i]);
+    if (hideElement.classList.contains('hide') === false) {
+      hideElement.classList.add('hide');
+    }
+  }
+};
```
Comment (AmaduKamara) [inline index.js]: - [ ] What if you could move these functions to a separate file respectively so you modularize your code and then import and use them in this file? Kindly work on that so this file is lean and your code is divided into reasonable modules. 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Adams-Ghub,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] The app does not seem to be working at the moment. Kindly work on making all links work fine and all other features work as expected 👍🏽
- [ ] You could add more modules by creating separate files for some functions and then importing and using them in the index.js file 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by OyePriscilla — APPROVED** [review]
Hi @Adams-Ghub 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Use ES6 modules to write modular JavaScript ✔️ 
Navigation links run as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**